build.sh

```diff
@@ -21,7 +21,9 @@ subproject_setup() {
 
 subproject_build() {
   pnpm install --frozen-lockfile
-  pnpm -r build
+  pnpm --recursive --filter "!./packages/stash-typedoc" build
+  # Build stash-typedoc last because it depends on the other packages already being built
+  pnpm --filter "./packages/stash-typedoc" build
```

Comment:
If you made all the packages dev dependencies it would evaluate in the correct order. Not a big deal though.

Reply:
Nice, that sounds like a better solution. I'll fix that up.